src/Abp.ZeroCore/Authorization/Roles/AbpRoleStore.cs

```diff
@@ -308,9 +315,8 @@ public virtual async Task<IList<Claim>> GetClaimsAsync([NotNull] TRole role, Can
 
             Check.NotNull(role, nameof(role));
 
-            await _roleRepository.EnsureCollectionLoadedAsync(role, u => u.Claims, cancellationToken);
-
-            return role.Claims.Select(c => new Claim(c.ClaimType, c.ClaimValue)).ToList();
+            return await _asyncQueryableExecuter.ToListAsync(_roleClaimRepository.GetAll()
+                .Where(x => x.RoleId == role.Id).Select(c => new Claim(c.ClaimType, c.ClaimValue)));
```

Comment:
Yes, this method is not needed, I just replace the `EnsureCollectionLoaded `with a repository in the service just like the default implementation of identity.

Reply:
That is because [`IdentityRole`](https://github.com/dotnet/aspnetcore/blob/v3.1.1/src/Identity/Extensions.Stores/src/IdentityRole.cs) doesn't have the `Claims` collection property.

Since [`AbpRole`](https://github.com/aspnetboilerplate/aspnetboilerplate/blob/v5.1.0/src/Abp.ZeroCore/Authorization/Roles/AbpRole.cs) does and an instance of it is passed in, we should populate that property.